Config/BuildSettings.swift

```diff
@@ -427,4 +427,9 @@ final class BuildSettings: NSObject {
         // Do not enable live location sharing atm
         return false
     }
+    
+    // MARK: - Presence
+    
+    /// A setting to enable the presence configuration settings section.
+    static let presenceSettingsAllowConfiguration: Bool = false
```

Comment:
Can we have the prefix `settingsScreen` for this setting, as we have for others for the Settings screen.